nautobot/ipam/models.py

```diff
@@ -506,14 +506,15 @@ class Prefix(PrimaryModel, StatusModel, RoleModelMixin):
     class Meta:
         ordering = (
             "namespace",
+            "ip_version",
             "network",
             "prefix_length",
         )
         index_together = [
-            ["network", "broadcast", "prefix_length"],
-            ["namespace", "network", "broadcast", "prefix_length"],
+            ["ip_version", "network", "broadcast", "prefix_length"],
+            ["namespace", "ip_version", "network", "broadcast", "prefix_length"],
         ]
-        unique_together = ["namespace", "network", "prefix_length"]
+        unique_together = ["namespace", "ip_version", "network", "prefix_length"]
```

Comment:
We might want to be cautious about the the `index_together` and `unique_together` that share the same fields since they are both effectively indexing the same fields (unique just adds the constraint as well). The ordering is significant as well, so I think we should probably profile these assuming that you didn't already do this...

Reply:
Haven't done any performance profiling - just using the `generate_test_data` command for now which is where I saw the ordering issues.

Reply:
Reverted the index_together/unique_together changes as I'm reasonably sure they're unnecessary.